### PR TITLE
SPCTR-2999 [Dynamic Content] Added Context as an attribute for use inside loop builder

### PR DESCRIPTION
**Pull request by abhishekrijal**
### Description
[Dynamic Content] Added Context as an attribute for use inside loop builder
The Attribute data is then being used as a source of postID and Post Type to fetch post specific custom meta keys inside Dynamic Content selection dropdown.

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/advanced-heading/edit.js [hunk 1]
```diff
@@ -34,6 +35,7 @@ const UAGBAdvancedHeading = ( props ) => {
 		// Assigning block_id in the attribute.
 		setAttributes( { block_id: clientId.substr( 0, 8 ) } );
 		setAttributes( { classMigrate: true } );
+		setAttributes( { context } );
```
Comment (rahulvarma722) [inline src/blocks/advanced-heading/edit.js:38]: Should we check the context in the attribute and based on changing context we should setAttribute context?

File: src/blocks/buttons-child/edit.js [hunk 2]
```diff
@@ -35,6 +36,8 @@ const ButtonsChildComponent = ( props ) => {
 		// Assigning block_id in the attribute.
 		setAttributes( { block_id: clientId.substr( 0, 8 ) } );
 
+		setAttributes( { context } );
```
Comment (rahulvarma722) [inline src/blocks/buttons-child/edit.js:39]: Should we check the context in the attribute and based on changing context we should setAttribute context?
Also, check in other blocks also this instance.

**Review by rahulvarma722 — CHANGES_REQUESTED** [review]

**Comment by mohitsbsftester** [comment]
Is there any backward compatibility to be checked in this PR?

File: src/blocks/buttons-child/edit.js (hunk 2 above)
Comment (abhishekrijal) [reply src/blocks/buttons-child/edit.js]: @rahulvarma722 Updated with check for context change.

File: src/blocks/advanced-heading/edit.js (hunk 1 above)
Comment (abhishekrijal) [reply src/blocks/advanced-heading/edit.js]: @rahulvarma722 Updated with check for context change.

**Pull request merged**